Readme/automation.md

```diff
@@ -0,0 +1,159 @@
+# Running Automation test with Kobiton with TeamCity
+## Table of contents
++ [A. Preparation](#a-preparation)
+  + [1. Prerequisites](#1-prerequisites)
+  + [2. Gettting Kobiton username and API key](#2-gettting-kobiton-username-and-api-key)
+  + [3. Samples](#3-samples)
++ [B. Setup](#b-setup)
+  + [1. Getting desired capabilities](#1-getting-desired-capabilities)
+  + [2. Configuring automation test script](#2-configuring-automation-test-script)
+  + [3. Configuring HockeyApp Webhooks listenner](#3-configuring-hockeyapp-webhooks-listenner)
+  + [4. Configuring TeamCity project](#4-configuring-teamcity-project)
++ [C. Execution](#c-execution)
+  + [1. Runnig automation test on Kobiton devices](#1-running-automation-test-on-kobiton-devices)
+  + [2. Fetch test session data through Kobiton REST API](#2-fetch-test-session-data-through-kobiton-rest-api)
++ [D. Feedback](#d-feedback)
+## A. PREPARATION
+### 1. Prerequisites
++ A server for receiving HockeyApp webhooks with Ruby 2.x installed.
+  > If you haven't installed Ruby on your server, please refer to [Ruby Installation documentation](https://www.ruby-lang.org/en/documentation/installation/) on how to install Ruby on corresponding operation system.
++ Install nodejs: Nodejs is required to run our sample automation test below.
```

Comment:
Install NodeJS ? I suppose it should be "NodeJS installed" or "NodeJS 6.x or higher installed"